lib/Scheduler/graph_trans.cpp

```diff
@@ -2,11 +2,71 @@
 #include "opt-sched/Scheduler/bit_vector.h"
 #include "opt-sched/Scheduler/logger.h"
 #include "opt-sched/Scheduler/register.h"
+#include "llvm/ADT/ArrayRef.h"
 #include "llvm/ADT/STLExtras.h"
+#include "llvm/ADT/SmallVector.h"
+#include <algorithm>
+#include <vector>
 #include <list>
 
 using namespace llvm::opt_sched;
 
+namespace {
```

Comment:
Can these just be static instead of inside an autonomous namespace.

Reply:
Yes, they can. But why do we prefer static functions to anonymous namespace functions?

Reply:
It's just a stylistic preference. https://llvm.org/docs/CodingStandards.html#anonymous-namespaces
I agree types are the obvious exception.

Reply:
Thanks, I didn't know it was in the Coding Standards.